app/vmalert/remotewrite/init.go

```diff
@@ -32,6 +32,8 @@ var (
 	maxBatchSize  = flag.Int("remoteWrite.maxBatchSize", 1e3, "Defines defines max number of timeseries to be flushed at once")
 	concurrency   = flag.Int("remoteWrite.concurrency", 1, "Defines number of writers for concurrent writing into remote querier")
 	flushInterval = flag.Duration("remoteWrite.flushInterval", 5*time.Second, "Defines interval of flushes to remote write endpoint")
+	retryCount    = flag.Int("remoteWrite.retryCount", 5, "Defines the number of attempts to write to the remote write endpoint")
```

Comment:
do you think we need to expose these settings? Under which circumstances would you change them and to what values?

Reply:
I dont mind exposing them in case that the defaults dont work for someone. I would change them if i expect unavailability > 5s in my infrastructure i guess. I also dont mind dont exposing them, what is your preference?

Reply:
I'd prefer not to expose extra cmd-line flags to keep it less complex. It would be easy to add them in the future if indeed, users will find it important.

Reply:
got rid of the command line arguments